app/web/src/components/EmbedProjectCell/EmbedProjectCell.tsx

```diff
@@ -0,0 +1,71 @@
+import useUser from 'src/helpers/hooks/useUser'
+import { useIdeState } from 'src/helpers/hooks/useIdeState'
+import { IdeContext } from 'src/helpers/hooks/useIdeContext'
+import EmbedViewer from '../EmbedViewer/EmbedViewer'
+
+export const QUERY = gql`
```

Comment:
You should be able to import than re export QUERY from `app/web/src/components/IdeProjectCell/IdeProjectCell.tsx` instead of redefining it here (can import the Project type)
I don't mind having some duplication, but having more than one query with the same name `FIND_PROJECT_BY_USENAME_TITLE` can cause issues with apollo, if you don't want to import the query could you atleast rename to something unique.

Reply:
Ah nice thank you, complete.